src/components/JoinUs/elements.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+import styled from 'styled-components'
+import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import { Grid, Row, Col } from '../grid'
+import { H2, H3, H5, Paragraph } from '../Typography'
+import GreyBG from '../GreyBG'
+import Hr from '../Common/Hr'
+
+/**
+ * Layout
+ */
+export const FirstColumn = ({ children }) => (
+  <Col width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>{children}</Col>
+)
+
+export const ShiftedColumn = styled(Col)`
+  ${breakpoint('tablet')`
+    margin-left: 8.33333%;
+  `};
+`
+export const SecondColumn = ({ children }) => (
+  <ShiftedColumn width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>{children}</ShiftedColumn>
+)
+
+export const List = ({ list }) => (
+  <Padding top={1}>
+    <ReactMarkdown
+      renderers={{
+        heading: props => <H5 as="h3" noMargin bold {...props} />, // eslint-disable-line react/display-name
+        paragraph: props => <Paragraph muted {...props} /> // eslint-disable-line react/display-name
+      }}
+      source={list}
+    />
+  </Padding>
+)
+
+export const Section = ({ children, greyBg, ...props }) => {
```

Comment:
This is nice :+1: I reckon we should find a consistent way to do this across the codebase?

If we're going to call it `Section`, that's a pretty good indication that it should be an HTML5 `section` under the hood. I reckon this would actually be a good idea across the whole website, by [changing `Grid` to always be section, instead of a div](https://github.com/yldio/yld.io/blob/master/src/components/grid.js#L110)?

Similarly, I would probably also pop a `backgroundColor` prop onto the Grid component (as opposed to just `greyBg` here). Maybe we could then create a utility function that applies the appropriate colour from the theme, as the background colour, & use this pattern in our `Header` (aka NavBar) as well?

Reply:
I called it section because it's inside the 'scope' of the page, so it's a section of the page. If this was to be moved for a `common` folder, then yes it should reflect the tag element name (or letting it pass an `as` prop). 

Regarding changing grid to always be a section, I'm not really sure if it will always make sense semantically (if for some reason we need inner grids - don't know if this will ever happen though). Our `background` components already render a `section` btw.

If this was/will be a common component, then yes a backgroundColor might make sense — for the purpose of this page, a simple bool was enough